src/utils/roundFloats.js

```diff
@@ -18,9 +18,9 @@ under the Apache 2.0 license is conditioned upon your compliance with
 such restriction.
 */
 export const roundFloats = (value, precision) => {
-  const floatNumber = parseFloat(value)
+  if (isNaN(value)) return value
```

Comment:
This logic isn't correct. It doesn't handle the case if value `1password`. This value isn't NaN and it will return just a number 1

Reply:
in case of "1password" it should return "1password" as a string.

![ML-5455-5-test-roundFloat-func](https://github.com/pinis-gini-apps/ui/assets/97065270/ad4103d3-6499-4cc2-be82-a072061de807)

Reply:
![Uploading image.png…]()

Reply:
@mavdryk  i'm update the logic. screenshot is attached.

![Uploading ML-5455-roundFloat-v2.png…]()

Reply:
![Uploading ML-5455-roundFloat-v2.png…]()